src/core/retargeting_engine_tests/python/test_sources.py

```
@@ -221,6 +222,45 @@ def test_head_source_outputs(self):
         assert "head" in output_spec
         assert len(output_spec) == 1
 
+    def test_head_source_compute_active(self):
+        """Test that HeadSource converts active tracked data correctly."""
+        source = HeadSource(name="head")
+
+        head_data = HeadPoseT(
+            Pose(Point(1.0, 2.0, 3.0), Quaternion(0.0, 0.0, 0.0, 1.0)),
+            True,
```

Ruff (0.15.2)

[error] Boolean positional value in function call

(FBT003)